circle.yml

```diff
@@ -61,6 +61,11 @@ test:
         parallel: true
     - codecov:
         parallel: true
-    - rm -rf docs/API && make -C docs clean
+    - pyenv local 3.4.3:
+        parallel: true
+    - pip install -r docs-requirements.txt:
+        parallel: true
+    - rm -rf docs/API && make -C docs clean:
+        parallel: true
     - python setup.py docs:
         parallel: true
```

Comment:
@jayvdb I meanwhile chose a similar approach by just setting `pyenv local 3.4.3` before building docs:

```
    - pyenv local 3.4.3:  # sphinx fails with 3.5.1
        parallel: true
    - pip install $(ls ./dist/*.whl)"[alldeps]":
        parallel: true
    - pip install -r docs-requirements.txt:
        parallel: true
    - rm -rf docs/API && make -C docs clean:
        parallel: true
    - python setup.py docs:
        parallel: true
```

The additional `pip install $(ls ./dist/*.whl)"[alldeps]"` is almost no overhead because all requirements are already installed at that point